exercises/practice/secret-handshake/hints/hints.md

```diff
@@ -1,20 +0,0 @@
-use Bitwise (or div/rem)
-
-If you use Bitwise, an easy way to see if a particular bit is set is to compare
-the binary AND (`&&&`) of a set of bits with the particular bit pattern you
-want to check, and determine if the result is the same as the pattern you're
-checking.
-
-Example:
-
-Flags: 0b11011
-Check: 0b11010
-
-Flags &&& Check: 0b11010 (All checked bits are set)
-
-Another:
-
-Flags: 0b11011
-Check: 0b10110
-
-Flags &&& Check: 0b10010 (Third bit not set)
```

Comment:
I don't know how to format this whole text as a list item that would fit on the website. AFAIK sublists and code blocks are not allowed in hints.

Reply:
I'm not sure actually. I _think_ our only requirement is for the hints to be in a list. I don't know for sure though.

Reply:
I had to remove sublists from concept exercise hints before because they would break the website.

Reply:
Ah okay, I didn't know. Sorry